server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -151,9 +153,10 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
 
     @Override
     protected Settings nodeSettings(int nodeOrdinal) {
-        if (segmentRepoPath == null || translogRepoPath == null) {
+        if (segmentRepoPath == null || translogRepoPath == null || clusterStateRepoPath == null) {
             segmentRepoPath = randomRepoPath().toAbsolutePath();
             translogRepoPath = randomRepoPath().toAbsolutePath();
+            clusterStateRepoPath = randomRepoPath().toAbsolutePath();
```

Comment:
I didnt get this change, we wanted to have separate repo for remote routing table. Cluster state repo is already being initialized using the segment repo details.